### ISSUE TITLE
Re-run models with updated dummy coding

### ISSUE THREAD

**Issue by marcdotson** [andrewheiss/why-donors-donate#19]
What do we need in terms of model comparison to test the specified hypotheses?

- There are a variety of ways we could measure in-sample fit.
- As a point of discussion: now that I know how to implement loo for a hierarchical model, we may need to re-run the models so we can use that for model comparison.

**Comment by marcdotson** [comment]
To update:

- Validate the parameterization.
- Consider any recoding of the covariates.
- Re-run the models and include generated quantities for PSISI-LOO measure of model fit.
- Compute contrasts instead of eyeballing comparisons between parameter estimates.

**Comment by marcdotson** [comment]
Models are running again.

**Comment by marcdotson** [comment]
@andrewheiss in lieu of a premature PR, I'm reopening this issue with some updates to guide what you see in the `model-rerun` branch if you get a chance to look at it early next week.

- I'm using a new parameterization of the model that I'm more comfortable with (see [this blog post](https://www.occasionaldivergences.com/post/choice-models/) for details). I don't see it as materially changing the results.
- I've also updated the covariates coding so _everything_ is dummy-coded in the updated `02_clean-data.Rmd`. That means I made up names for some of the anchors in rating scales, but overall I think it makes evaluating the posterior marginals a bit more straightforward and feeds into the market simulator better. I've also changed the order of the covariates so they consistently go from less to more, wherever that makes sense, in an attempt to make looking at the results more consistent.
- The new model code also includes model fit information. Calibration and model fit are now detailed in the updated `03_analysis.Rmd` with the relevant chunks set to `eval=FALSE` (though I still can't knit that R Markdown file for whatever reason). The output of the model comparison is one of those details. It might be informative and worth turning into an actual table in the paper.

I do have a few outstanding questions.

- In terms of actual plots and tables to include in the paper, I'm assuming that's the organizing principle around `04_figs-tables.Rmd`. I haven't moved anything into that R Markdown, but I have collapsed the other analysis R Markdown files all into `03_analysis.Rmd`. I'm sure much of it isn't particularly useful beyond just diagnostics, so feel free to prune as needed. I'll keep working on that.
- I think contrasts, which we can create by just computing the difference between two sets of draws, might be a cleaner way to answer some of the hypotheses.
- As noted above, the model fit table should probably make its way into `04_figs-tables.Rmd` if you think it's useful. I've saved out the relevant data frame as `model_fit.rds` in `output`.

Let me know if you want to jump on a call to discuss anything with respect to the new model results. As we wrap this up along with the `market-simulator`, I'm ready for marching orders regarding sections in the paper itself.
